### DISCUSSION TITLE
blog post

### DISCUSSION THREAD

**Discussion by victorazangu** [SpaceyaTech/Team-Rio-Django#10] [General]
am working on a BlogPost
any help or additional ideas we are looking forward hearing from you

**Comment by mamu234** [comment]
Please refer to our previous discussion with Hudson on the features needed for the blogpost  .Anyone who wrote them down can share them

**Reply by sangkips** [reply]
User Attributes 
Unique ID(UUID)
Username(combine firstname and lastname)
Email address must be unique
Password
Phone number must be unique
Dsplay picture(image URL and assign a default avatar)
Gender
date created
date updated
account status
verification code
role_id
about(bio)

**Reply by sangkips** [reply]
Blogpost Attributes
Unique ID
post title
description
content
user_id
status of the post
date published
date updated 
post category(here we will create a class to hold categories)
image URLs
reactions 
views 
comments
